<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoMuchWoWApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>206</width>
        <height>251</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>535</width>
         <height>225</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Customers</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="verticalWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="text">
                  <string>Search filters</string>
                 </property>
                 <property name="maxLength">
                  <number>14</number>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="idLineEdit">
                 <property name="placeholderText">
                  <string>Id</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLineEdit" name="firstNameEdit">
                 <property name="placeholderText">
                  <string>First name</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lastNameEdit">
                 <property name="placeholderText">
                  <string>Last name</string>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QDateEdit" name="startDate">
                 <property name="dateTime">
                  <datetime>
                   <hour>0</hour>
                   <minute>0</minute>
                   <second>0</second>
                   <year>2018</year>
                   <month>1</month>
                   <day>1</day>
                  </datetime>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDateEdit" name="endDate">
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="dateTime">
                  <datetime>
                   <hour>0</hour>
                   <minute>0</minute>
                   <second>0</second>
                   <year>2019</year>
                   <month>1</month>
                   <day>1</day>
                  </datetime>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editTablePushButton">
             <property name="text">
              <string>Edit Table</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Staff list</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <widget class="QPushButton" name="editTreePushButton">
          <property name="text">
           <string>Edit tree</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Schedule</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="verticalWidget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QDateEdit" name="planDateEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="planPushButton">
               <property name="text">
                <string>Schedule</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextBrowser" name="planTextBrowser">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="QPushButton" name="savePushButton">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pathPushButton">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLineEdit" name="pathLineEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="saveLineEdit">
               <property name="placeholderText">
                <string>Filename</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionCustomer"/>
     <addaction name="actionStaff"/>
    </widget>
    <addaction name="menuAjouter"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCustomer"/>
   <addaction name="actionStaff"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCustomer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/customer.png</normaloff>:/Resources/customer.png</iconset>
   </property>
   <property name="text">
    <string>Customer</string>
   </property>
  </action>
  <action name="actionStaff">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/staff.png</normaloff>:/Resources/staff.png</iconset>
   </property>
   <property name="text">
    <string>Staff</string>
   </property>
   <property name="toolTip">
    <string>Staff</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/about.png</normaloff>:/Resources/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Resources/exit.png</normaloff>:/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
